argus/backend/service/user.py

```diff
@@ -90,7 +90,10 @@ def github_callback(self, req_code: str) -> dict | None:
         except User.DoesNotExist:
             user = User()
             user.username = user_info.get("login")
-            user.email = email_info[-1].get("email")
+            # pick only scylladb.com emails
+            scylla_email = next(iter([email.get("email") for email in email_info if email.get["email"].endswith("@scylladb.com")]), None)
```

Comment:
how about existing users? shouldn't we update their emails to conform this too?
E.g by verification if user.email ends with scylladb.com and if not try to reassign it like here? 
Btw. in case of no scylladb email, shouldn't we  abandon user?

Reply:
I don't know if we should drop or not, but already check if they belongs to the github organization

anyhow we could check the current state of user, and see if fixes are needed or not